docker/Dockerfile

```diff
@@ -16,18 +16,26 @@
 #
 
 # Usage:
-#   1. use ./build/dist to make binary distributions of Kyuubi or download a release
-#   2. Untar it and run the docker command below
-#      docker build -f docker/Dockerfile -t yaooqinn/kyuubi:tagname .
+#   Run the docker command below
+#      docker build --build-arg MVN_ARG="-Pmirror-cn" -f docker/Dockerfile -t apache/kyuubi:tagname .
```

Comment:
I think it's better to change the demo value of `MVN_ARG` from `-Pmirror-cn` to `-Pspark-3.1,spark-hadoop-3.2`

Reply:
make sense